pyspecdata/mpl_utils.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 from numpy import r_, pi
+from matplotlib import ticker
```

Comment:
Import ticker as mticker, and then do not make the change that you make below

Reply:
this comment has not been addressed

Reply:
edited